target/linux/ath79/dts/qca9558_allnet_all-wap02860ac.dts

```diff
@@ -95,88 +56,10 @@
 	};
 };
 
-&pcie0 {
-	status = "okay";
-
-	wifi@0,0 {
-		compatible = "qcom,ath10k";
-		reg = <0 0 0 0 0>;
-	};
-};
-
-&pll {
-	clocks = <&extosc>;
-};
```

Comment:
Why is this thrown away?

Reply:
this is already defined in qca955x.dtsi

Reply:
Then please address this in a separate commit (a comment would have helped ...)